plugins/inputs/smart/smart_test.go

```diff
@@ -379,6 +379,13 @@ func TestExcludedDev(t *testing.T) {
 }
 
 func TestGatherSATAInfo(t *testing.T) {
+	s := NewSmart()
```

Comment:
Nitpick: on these, just define inline:
```go
s := &Smart{
  Timeout: internal.Duration{Duration: time.Second * 30},
  UseSudo: true,
  etc...
}
```

Reply:
NewSmart() is setting default Timeout, maybe just remove setting Timeout in these tests and allow NewSmart() to do it ? It's going to be in line with other tests like: TestGatherAttributes, TestGatherNoAttributes.

Reply:
Makes sense, don't set Timeout so match those other tests, or update the other tests to define inline. I'm fine with either way.

Reply:
removed Timeout